packages/@aws-cdk/aws-s3/test/rules.test.ts

```diff
@@ -96,6 +96,38 @@ describe('rules', () => {
     });
   });
 
+  test('Bucket with no transition duration fails', () => {
```

Comment:
The rule of thumb is to also have integ tests accompany this kind of changes.

Reply:
sorry, AFAIK I cannot do an integ test for this.